tools/system_libs.py

```diff
@@ -728,6 +728,11 @@ def get_files(self):
           if not cancel:
             libc_files.append(os.path.join(musl_srcdir, dirpath, f))
 
+    # Allowed files from blacklisted modules
```

Comment:
Indentation off?

Reply:
I think it's ok now, maybe I fixed it in a merge from master meanwhile?